charts/generic-service/values.schema.json

```diff
@@ -937,6 +937,27 @@
                 "additionalProperties": false
               },
               "description": "Alerts for messages of specific log levels"
+            },
+            "lokiSideCar": {
+              "type": "object",
+              "properties": {
+                "configMapLabel": {
+                  "type": "string",
+                  "default": "loki_rules",
+                  "description": "The label attached to the configMap holding the Loki Rules."
+                },
+                "configMapLabelValue": {
+                  "type": "string",
+                  "default": "",
+                  "description": "The value of the label attached to the Loki Rule configMap."
+                }
+              },
+              "additionalProperties": false
+            },
+            "createLokiRule": {
+              "type": "boolean",
+              "default": true,
```

Comment:
In order for a default value to be effective, you'll also need to set it in `values.yaml`.